views/authed.twig

```diff
@@ -306,7 +306,7 @@
                 <option value="intArray" {{#eq type "intArray"}}selected="selected"{{/eq}}>intArray</option
             </select>
             <input class="form-control form-control-inline validated-element extra-value" placeholder="{% endverbatim %}{% trans "Value" %}{% verbatim %}" value="{{ value }}">
-            <button type="button" class="btn btn-sm btn-warning pull-right intent-remove-extra"><i class="fa fa-minus fa-plus"></i></button>
+            <button type="button" class="btn btn-sm btn-warning pull-right intent-remove-extra"><i class="fa fa-minus"></i></button>
```

Comment:
This one works, but I think you've missed the same fix in line 400 for the `command-input-http-key-value-template` template

Reply:
Thanks for pointing that